infermedica_api/connectors/__init__.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+
+"""
+infermedica_api.connectors
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This module contains API Connector classes and related utils.
+"""
+from typing import Union
+
+from .common import SearchConceptType
+from .v2 import BasicAPIv2Connector, APIv2Connector, ModelAPIv2Connector
+from .v3 import BasicAPIv3Connector, APIv3Connector, ConceptType
```

Comment:
`ConceptType` not used?

Reply:
Used in `infermedica_api/__init__.py` so one can do `from infermedica_api import ConceptType`.